imports/plugins/core/cart/server/no-meteor/registration.js

```diff
@@ -0,0 +1,17 @@
+// Objects with `name`, `priority` and `fn` properties
+export const cartTransforms = [];
+
+/**
+ * @summary Will be called for every plugin
+ * @param {Object} options The options object that the plugin passed to registerPackage
+ * @returns {undefined}
+ */
+export function registerPluginHandler({ cart }) {
+  if (cart) {
+    const { transforms } = cart;
+    if (Array.isArray(transforms)) {
+      cartTransforms.push(...transforms);
+      cartTransforms.sort((prev, next) => prev.priority - next.priority);
+    }
```

Comment:
This check just silently rejects a non-array? Should it create an array or throw an error?